src/nvim/api/private/helpers.c

```diff
@@ -1968,26 +1968,11 @@ bool parse_float_config(Dictionary config, FloatConfig *fconfig, bool reconf,
       }
       has_bufpos = true;
     } else if (!strcmp(key, "external")) {
-      if (val.type == kObjectTypeInteger) {
-        fconfig->external = val.data.integer;
-      } else if (val.type == kObjectTypeBoolean) {
-        fconfig->external = val.data.boolean;
-      } else {
-        api_set_error(err, kErrorTypeValidation,
-                      "'external' key must be Boolean");
-        return false;
-      }
-      has_external = fconfig->external;
+      has_external = fconfig->external
+          = api_object_to_bool(val, "'external' key", false, err);
     } else if (!strcmp(key, "focusable")) {
-      if (val.type == kObjectTypeInteger) {
-        fconfig->focusable = val.data.integer;
-      } else if (val.type == kObjectTypeBoolean) {
-        fconfig->focusable = val.data.boolean;
-      } else {
-        api_set_error(err, kErrorTypeValidation,
-                      "'focusable' key must be Boolean");
-        return false;
-      }
+      fconfig->focusable
+          = api_object_to_bool(val, "'focusable' key", false, err);
```

Comment:
As this defaults to `true`, should I pass `true` as the 3rd arg here?

Reply:
yes.